cadwyn/structure/versions.py

```diff
@@ -448,7 +448,10 @@ async def _convert_endpoint_response_to_version(
             path,
             method,
         )
-        if isinstance(response_or_response_body, FastapiResponse):
+        if isinstance(response_or_response_body, FastapiResponse) and response_info.body:
```

Comment:
We need to cover this case by a test that fails on the previous version.

Reply:
it cant be covered, because uvicorn still responses, but with a runtime error

Reply:
Sent you the tests  that cover it, though we really cannot make them fail, sadly

Reply:
added